{"threat":["A local attacker could possibly use this issue to execute arbitrary code.."],"info":"python-dbusmock vulnerability","date":"May 21, 2015","CVEs":["CVE-2015-1326"],"special":"The problem can be corrected by updating your system to the following package versions:\nIn general, a standard system update will make all the necessary changes.","update":{"14.10":[{"package":"python-dbusmock","version":"0.11.4-1ubuntu1"},{"package":"python3-dbusmock","version":"0.11.4-1ubuntu1"}],"14.04":[{"version":"0.10.1-1ubuntu1","package":"python-dbusmock"},{"version":"0.10.1-1ubuntu1","package":"python3-dbusmock"}],"15.04":[{"package":"python-dbusmock","version":"0.14-1ubuntu2"},{"package":"python3-dbusmock","version":"0.14-1ubuntu2"}]},"usn":"USN-2618-1"}